generation_config.yaml

```
@@ -1845,6 +1845,8 @@ libraries:
   GAPICs:
   - proto_path: google/cloud/secretmanager/v1
   - proto_path: google/cloud/secretmanager/v1beta2
+  # v1beta1's version is in a different proto path from the rest
+  - proto_path: google/cloud/secret/v1beta1
```

The hermetic build failed because the path does not exist, I think it should be 
```suggestion
  - proto_path: google/cloud/secrets/v1beta1
```
per https://github.com/googleapis/googleapis/tree/master/google/cloud/secrets/v1beta1

---

Thanks!

---

Updated in the yaml file as well